README.md

```diff
@@ -21,7 +21,7 @@ IMG=docker.io/${USER}/tempo-operator:dev-$(date +%s) make generate bundle docker
 
 3. Create a secret for minio in the namespace you are using:
 ```yaml
-kubectl apply --namespace ${NAMESPACE} -f - <<EOF
```

Comment:
This change is not related to the PR, but I found it rather not useful to have the `${NAMESPACE}` substitution in the quickstart.